spec/support/collections_publisher_helpers.rb

```diff
@@ -10,4 +10,18 @@ def fill_in_topic_form(slug:, title:, parent: nil)
     fill_in "Title", with: title
     fill_in "Description", with: sentence
   end
+
+  def self.included(base)
+    if SignonHelpers::use_signon?
```

Comment:
might be slightly cleaner to have a guard clause and do `return unless SignonHelpers::use_signon?`

Reply:
That works for me, I've updated all the support files.